### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hackit-coder**
Updated the link and corrected many grammatical and typos mistake.

### CODE REVIEW

**Comment by hackit-coder** [comment]
Plz review mods

**Comment by hackit-coder** [comment]
@thinkwee plz look

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
+    【 <a href="readme/README-English.md">English original</a> | <a href="readme/README-Chinese.md">Chinese</a> | <a href="readme/README-Japanese.md">Japanese</a> | <a href="readme/README-Korean.md">Korean</a> | <a href="readme/README-Filipino.md">Filipino</a> | <a href="readme/README-French.md">French</a> | <a href="readme/README-Slovak.md">Slovak</a> | <a href="readme/README-Portuguese.md">Portuguese</a> | <a href="readme/README-Spanish.md">Spanish</a> | <a href="readme/README-Dutch.md">Dutch</a> | <a href="readme/README-Turkish.md">Turkish</a> | <a href="readme/README-Hindi.md">Hindi</a> | <a href="readme/README-Bahasa-Indonesia.md">Bahasa Indonesia</a> | <a href="readme/README-Russian.md">Russian</a> | <a href="readme/README-Urdu.md">Urdu</a>】
```
Comment (thinkwee) [inline README.md:8]: Thank you, but there is no README-English.md under the ``readme/``, the English readme is the default README.md.

File: README.md [hunk 2]
```diff
@@ -15,73 +15,73 @@
...
+  
+•June 12, 2024: We introduced Multi-Agent Collaboration Networks (MacNet) 🎉, which utilize directed acyclic graphs to facilitate effective task-oriented collaboration among agents through linguistic interactions 🤖🤖. MacNet supports co-operation across various topologies and among more than a thousand agents without exceeding context limits. More versatile and scalable, MacNet can be considered as a more advanced version of ChatDev's chain-shaped topology. Our preprint paper is available at [https://arxiv.org/abs/2406.07155](https://arxiv.org/abs/2406.07155). This technique will soon be incorporated into this repository, enhancing support for diverse organizational structures and offering richer solutions beyond software development (e.g., logical reasoning, data analysis, story generation, and more).
```
Comment (thinkwee) [inline README.md:35]: •is not a standard markdown symbol

**Comment by thinkwee** [comment]
Thank you! Please see comments above.

**Comment by hackit-coder** [comment]
Sir, plz accept this, it's my first contribution in open source.

**Comment by thinkwee** [comment]
> Sir, plz accept this, it's my first contribution in open source.

Thank you. Also, could u please address the second comment? (* -> • is unnecessary)

**Comment by hackit-coder** [comment]
• is a bullet for better understanding 
The * is at the top of the word and it doesn't aligns with the sentences. We should use bullets to describe the updates. Hence

**Review by thinkwee — APPROVED** [review]

**Pull request merged**